src/settings.ts

```diff
@@ -16,7 +16,7 @@ export const SPARKLEVERSE_LOGO_URL = "/sparkleverse-logo.png";
 export const SPARKLE_LOGO_URL = "/sparkle-header.png";
 export const MEMRISE_LOGO_URL = "/memrise-logo.png";
 export const BURN_START_UTC_SECONDS = 1598770800; // Sunday Aug 30th, 2020 (easy to change later)
-export const VENUE_CHAT_AGE_DAYS = 10;
+export const VENUE_CHAT_AGE_DAYS = 30;
```

Comment:
sensible change and seems like a good idea; just wondering what motivated it?

Reply:
Ed, yesterday.